### DISCUSSION TITLE
How to remove Kasa HomeKit enabled accessories from HomeBridge?

### DISCUSSION THREAD

**Discussion by kjn70** [plasticrake/homebridge-tplink-smarthome#277] [General]
I'm using HomeBridge on a Synology to integrate Ring and Kasa accessories into HomeKit.  Some Kasa accessories are HomeKit enabled by default (SmartPlugs) but show up again through HomeBridge.

These accessories keep on showing up in HomeKit duplicated (once through the native Kasa HomeKit integration and once through HomeKit).  I would like to keep the native option and remove the HomeBridge option so that the only Kasa accessories HomeBridge shows are those that are not native HomeKit.

I've tried hiding the accessories in HomeBridge but that only appeared to hide them there.  They still show up in Apple Home.  I also tried to remove the cache through HomeBridge settings but as they are active they get added back.

Is there a way to permanently disable an accessory so that HomeBridge does not pass it across to Apple Home?

**Comment by gkondylas** [comment]
Go to homebridge settings from the UI, then go to Remove Single Cached Accessory and from that list find the plug you want to remove. From my list the ones that I had, the 4 last characters of the plugs were matching with the 4 letters of the serial number of the device. You can also see that serial number under the accessories tab in homebridge and then select the accessory you are looking for. I had to exclude them one by one in order to find it as the cashed plug that was removed from my home didn't had in the cashed accessories list the 4 characters of the serial number.
